package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@communicatehealth/skiplink.js",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
You can leave this change out — npm versioning process after merge will increment this value and the version in the lockfile

Reply:
Change to package version has been reverted.